rero_ils/modules/documents/utils.py

```diff
@@ -245,8 +245,25 @@ def title_format_text_head(titles, responsabilities=None, with_subtitle=True):
             if len(title_texts) == 1:
                 head_titles.append(title_texts[0].get('value'))
             else:
+                languages = [title.get('language') for title in title_texts]
+
+                def filter_list(value):
```

Comment:
I don't completely understand what this function is doing, more comments would be welcome.